repose-aggregator/components/filters/regex-rbac-filter/src/test/scala/org/openrepose/filters/regexrbac/RegexRbacSchemaTest.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.filters.regexrbac
+
+import org.junit.runner.RunWith
+import org.openrepose.commons.test.ConfigValidator
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.junit.JUnitRunner
+import org.xml.sax.SAXParseException
+
+@RunWith(classOf[JUnitRunner])
+class RegexRbacSchemaTest extends FunSpec with Matchers {
```

Comment:
Thought we ended up with a base class for these? Did that not work out?

Reply:
Not that I'm aware of. We start with the same basic format, but then have specific tests for each schema.

Reply:
`ConfigurationTest` is the class you are looking for. That would be the "base".

Reply:
Updated.

Reply:
A story was opened to convert the all the others:

* https://repose.atlassian.net/browse/REP-6337